# Message containing the camera head orientation
float32 yaw
float32 pitch

